client/components/gravatar/index.jsx

```diff
@@ -64,10 +73,18 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		const avatarURL = this.props.tempImage ||
+			this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
 
 		return (
 			<img alt={ alt } className="gravatar" src={ avatarURL } width={ size } height={ size } onError={ this.onError } />
 		);
 	}
-} );
+}
+
+export default connect( ( state, ownProps ) => {
+	return {
+		tempImage: getUserTempGravatar( state, get( ownProps, 'user.ID' ) ),
```

Comment:
Okay, I see.

The idea I have is to have some default value the `get` helper fn returns if `user.ID` is not found in `ownProps`. It would be a bit easier to reason about if I saw the default value there.

In case `getUserTempGravatar` accepts `false` as the `userId`, we can just add it like:

```js
tempImage: getUserTempGravatar( state, get( ownProps, 'user.ID', false ) )
```